src/components/DisplayTaxonName.js

```diff
@@ -71,9 +71,12 @@ const DisplayTaxonName = ( {
   return (
     <View
       testID="display-taxon-name"
-      className={classNames( "flex", null, {
-        "flex-row items-end flex-wrap w-11/12": isHorizontal
-      } )}
+      // 03032023 amanda - it doesn't look to me like we need these styles at all,
+      // and they're making the common name and sci name show up on the same
+      // line. not sure if i'm missing context here
+      // className={classNames( "flex", null, {
+      //   "flex-row items-end flex-wrap w-11/12": isHorizontal
+      // } )}
```

Comment:
no, just confused because the goal of this PR was to make the UI closer to what's in Figma, and two names on the same line is not what Abhas has for a 2-name design